esmvalcore/cmor/table.py

```diff
@@ -34,40 +35,181 @@ def read_cmor_tables(cfg_developer=None):
         with cfg_file.open() as file:
             cfg_developer = yaml.safe_load(file)
 
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    alias_file = os.path.join(cwd, 'variable_alias.yml')
```

Comment:
Would it be possible to change `alias` to something else? Because you also have alias defined here, with a very different purpose and it's a bit confusing like this: https://github.com/ESMValGroup/ESMValCore/blob/b7c5866b5695674043ed49ed0255d828b26f7005/esmvalcore/_recipe.py#L1079-L1081

Reply:
alt_names maybe?